scripts/ench_books.zs

```diff
@@ -0,0 +1,37 @@
+
+// Efficiency 5
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:dustRedstone> * 16)
+    .outputs(<minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 5 as short, id: 32 as short}]}))
+    .duration(100)
+    .EUt(14)
+    .buildAndRegister();
+
+// Unbreaking 3
+assembler.recipeBuilder()
+    .inputs(<ore:paper> * 48)
+    .inputs(<ore:ingotInvar> * 16)
```

Comment:
Ingots is definitely the wrong way to go. Use plates. Additionally Invar is already a heavily used material in the EBF. I would like to suggest Brass here since it uses the lesser used Zinc at this point in the game.